site/public/js/rainbow-customization.js

```diff
@@ -271,6 +271,100 @@ function getGradeableBuckets() {
     return gradeables;
 }
 
+function getPlagiarism() {
+    const plagiarismData = [];
+
+    const tableBody = document.getElementById('table-body');
+    const rows = tableBody.getElementsByTagName('tr');
+
+    for (let i = 0; i < rows.length; i++) {
+        const row = rows[i];
+        const user = row.cells[0].innerHTML;
+        const gradeable = row.cells[1].innerHTML;
+        const penalty = parseFloat(row.cells[2].innerHTML);
+
+        plagiarismData.push({
+            user: user,
+            gradeable: gradeable,
+            penalty: penalty,
+        });
+    }
+
+    return plagiarismData;
+}
+
+// eslint-disable-next-line no-unused-vars
+function addToTable() {
+    const USERID = document.getElementById('user_id').value.trim();
+    const gradeable = document.getElementById('g_id').value.trim();
+    const penalty = document.getElementById('marks').value.trim();
+
+    // Check for empty fields
+    if (USERID === '' || gradeable === '' || penalty === '') {
+        alert('Please fill in all the fields fyi.');
```

Comment:
Keep it concise and professional please.

```suggestion
        alert('Please fill in all the fields.');
```